http/http-advanced-reactive/src/main/resources/application.properties

```diff
@@ -67,3 +67,5 @@ pl-container-request-filter.enabled=false
 # Register MultipartFormDataReader as provider (used by io.quarkus.ts.http.advanced.reactive.MultipartResource.multipartFormData)
 quarkus.index-dependency.resteasy-multipart.group-id=org.jboss.resteasy
 quarkus.index-dependency.resteasy-multipart.artifact-id=resteasy-multipart-provider
+
+quarkus.qe.test.value=42
```

Comment:
@fedinskiy @pjgg 

I know we are Quarkus QE :-) but using `quarkus.qe` property prefix goes directly against docs https://quarkus.io/guides/config-reference#configuring_quarkus : _As mentioned above, properties prefixed with quarkus. are effectively reserved for configuring Quarkus itself and its extensions. Therefore, the quarkus. prefix should never be used for application specific properties._